### PR TITLE
feat: Add trunk tests

### PR DESCRIPTION
**Pull request by fxisco**
and Change car configuration to have an object for flexibility

## Review checklist

- [ ] Add the corresponding label
- [ ] Would you be proud of this work? 

## Testing
- Test that web page is working fine.
- Tests should work fine.
```shell
$ npm run test
```

Steps to test the change:

## Deploy checklist

- [ ] Rebase and squash your branch so it is one commit ahead of `origin/master`
- [ ] Use your PR title and the entire summary above the line as the commit message
- [ ] Delete branch after merging

### CODE REVIEW

File: test/Car.test.js [hunk 1]
```diff
@@ -53,5 +61,73 @@ describe('Car', () => {
...
     });
+
+    test('add items to trunk', () => {
+      maitee.toggleTrunkState();
+
+      maitee.addToTrunk({});
+      expect(maitee.getTrunkItems()).toEqual([]);
+
+      maitee.addToTrunk(null);
+      expect(maitee.getTrunkItems()).toEqual([]);
```
Comment (manuelamercado) [inline test/Car.test.js]: Do you think is better to use Chai? http://chaijs.com/api/bdd/ (Suggested by @sjortiz)

Comment (fxisco) [reply test/Car.test.js]: Could you confirm if the suggestion is to remove jest and use Chai or use Chai together with jest?

Comment (manuelamercado) [reply test/Car.test.js]: Chai with jest

Comment (fxisco) [reply test/Car.test.js]: Hey @manuelamercado could we have the chai implementation in a separate commit? Need to change all previous changes. Don't want to merge those changes with this so the commits stay clear.

Comment (manuelamercado) [reply test/Car.test.js]: Ok, no problems @fxisco

**Pull request merged**